KenticoListApp/TodoList.Api/Models/ListItem.cs

```diff
@@ -1,28 +1,25 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel.DataAnnotations;
-using System.Linq;
-using System.Web;
 
 namespace TodoList.Api.Models
 {
     public class ListItem
     {
-        public ListItem(string id, string text)
+        public ListItem(Guid id, string text)
         {
             Id = id;
             Text = text;
         }
 
-        [Required]
-        public string Id { get; set; }
+        public ListItem(Guid id)

```

Comment:
This one should call the other if you need it. Question is if you want to have special constructor just because of tests.